src/main.ts

```diff
@@ -3,12 +3,16 @@ import { NestApplication, NestFactory } from '@nestjs/core';
 import { MainModule } from '@ukef/main.module';
 
 import { App } from './app';
+import { REDACT_STRINGS } from './constants';
+import { ConsoleLoggerWithRedact } from './logging/console-logger-with-redact';
 
 const main = async () => {
-  const nestApp: NestApplication = await NestFactory.create(MainModule, { bufferLogs: true });
+  // If REDACT_LOGS is true use ConsoleLoggerWithRedact. ConsoleLoggerWithRedact is used just if `NestFactory.create` fails completely.
+  // If `NestFactory.create` doesn't fail completely, then buffered logs are passed to PinoLogger. NestLogger and ConsoleLoggerWithRedact are not used.
+  const logger = process.env.REDACT_LOGS !== 'false' ? new ConsoleLoggerWithRedact(REDACT_STRINGS) : new NestLogger();
```

Comment:
isn't this `boolean`? rather than `string`?

Reply:
All variables in process.env are strings. We need to parse them to boolean and numbers.

Usually we get configuration from Nestjs configService, but it is not available yet in this place, configuration service would return correct type `boolean`.